spec/models/declaration_spec.rb

```diff
@@ -192,6 +192,24 @@
     end
   end
 
+  describe "scopes" do
+    describe ".completed" do
+      it "returns completed declarations" do
+        FactoryBot.create(:declaration, declaration_type: "started")
+        completed_dec = FactoryBot.create(:declaration, declaration_type: "completed")
+        expect(described_class.completed.to_a).to eq([completed_dec])
```

Comment:
Just a suggestion take it or leave it 🙂  `expect(described_class.completed).to contain_exactly(completed_dec)`